### DISCUSSION TITLE
Allow gibs to stay on screen longer with a graphics setting

### DISCUSSION THREAD

**Discussion by poperigby** [redeclipse/base#1170] [Ideas and Feature Requests]
The gibs only stay on screen for a few seconds, before shrinking away. For people with more powerful computers, a graphics setting to increase this duration would be helpful.

**Comment by qreeves** [comment]
`/gibfade <milliseconds>` (defaults to 5000 ms)

**Comment by poperigby** [comment]
Ah. Will this be added to the options menu in the future?

**Comment by qreeves** [comment]
Possibly, though if we added every variable like this that RE has, the list would be a mile long.

**Comment by poperigby** [comment]
This feels like a somewhat important one though, right?

**Comment by qreeves** [comment]
About as important as the other 1000 just like it.

**Comment by poperigby** [comment]
Heh. Good point. I'll close this for now. You can re-open if you want to add it to the settings later.

**Comment by cloudziq** [comment]
before making a new topic, you can ask/confirm on discord about some things

**Comment by poperigby** [comment]
Ok. My bad.

**Comment by qreeves** [comment]
It's OK to do this here on `world`, because that's why we have a separate repository for this. I don't really care where people ask a questions, just don't do it on other repositories (like `base`, etc) unless it's a confirmed issue or bug.

I don't generally close issues here either, unless we're looking to bury them (because they're pointless or sensitive topics).

**Comment by poperigby** [comment]
Got it. Thank you.
